CHANGELOG.md

```diff
@@ -0,0 +1,128 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## [0.0.2](https://github.com/energywebfoundation/iam-cache-server/pull/156) (2021-09-16)
+
+### Features
+
+- **app:** migrate from dgraph to postgres ([7be1437](https://github.com/energywebfoundation/iam-cache-server/commit/7be1437c08e75d913a49c1152058ea891766834b))
+- **app:** migrate from dgraph to postgres ([2591cf9](https://github.com/energywebfoundation/iam-cache-server/commit/2591cf9bb6bdbf4d6e5efce3367f2db4e453fa17))
```

Comment:
Are all of these new changes? I think the `volta` cache-server is already using postgres for example 🤔

Reply:
@jrhender this was auto generated so it generated it based on our commits but I added the Breaking Changes part myself which was why I pushed this so we can pick what's needed and not.